apps/settings/test/unit/panels/addon_details/addon_details_test.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+/* globals loadBodyHTML, MockMozApps, MockL10n */
+
+requireApp('settings/shared/test/unit/load_body_html_helper.js');
+require('/shared/test/unit/mocks/mock_l10n.js');
+require('mock_moz_apps.js');
+
+suite('Addon Details Test > ', function() {
+
+  var modules = [
+    'panels/addon_details/addon_details',
+    'shared_mocks/mock_manifest_helper',
+    'unit/mock_addon_manager'
+  ];
+  var map = {
+    '*': {
+      'shared/manifest_helper': 'shared_mocks/mock_manifest_helper',
+      'modules/addon_manager': 'unit/mock_addon_manager'
+    }
+  };
+  var MockAddon = {
+    observe: sinon.stub(),
+    unobserve: sinon.stub(),
+    instance: {
+      manifest: {}
+    }
+  };
+  var MockObsoleteAddon = Object.assign({}, MockAddon, {
+    instance: { manifest: { customizations: [] } }
+  });
+  var realL10n, realMozApps;
+
+  suiteSetup(function(done) {
+    loadBodyHTML('_addon_details.html');
+    // Create a new requirejs context
+    var requireCtx = testRequire([], map, function() {});
+    var that = this;
+    realL10n = navigator.mozL10n;
+    navigator.mozL10n = MockL10n;
+    realMozApps = navigator.mozApps;
+    navigator.mozApps = MockMozApps;
+
+    requireCtx(modules, function(AddonDetails, MockManifestHelper,
+      MockAddonManager) {
```

Comment:
use `(AddonDetails, MockManifestHelper, MockAddonManager) => {}` and remove `that`